R/pkg/R/utils.R

```diff
@@ -440,19 +446,23 @@ processClosure <- function(node, oldEnv, defVars, checkedFuncs, newEnv) {
         # Namespaces other than "SparkR" will not be searched.
         if (!isNamespace(func.env) ||
               (getNamespaceName(func.env) == "SparkR" &&
-              !(nodeChar %in% getNamespaceExports("SparkR")))) {  # Only include SparkR internals.
+              !(nodeChar %in% getNamespaceExports("SparkR")))) {
```

Comment:
Ah, right. The `nodechar` check is still in the proper place. How about `(getNamespaceName`? Should it be aligned with `!isNamespace` ?